specs/binding-kafka.spec/src/main/scripts/io/aklivity/zilla/specs/binding/kafka/schema/kafka.schema.patch.json

```diff
@@ -339,18 +339,37 @@
                             "enum": [ "none", "json_patch" ],
                             "deprecated": true
                         },
-                        "headers":
+                        "transforms":
```

Comment:
Make sure you have caught up to latest develop, I think this change is already present there.